routes/apply_discount.js

```diff
@@ -232,6 +234,65 @@ const responseSchema = {
   'type': 'object',
   'additionalProperties': false,
   'properties': {
+    'buy_together': {
+      'type': 'array',
+      'maxItems': 20,
+      'product_kit_discounts': {
```

Comment:
Isso aqui não é um JSON schema válido

Reply:
```suggestion
    'buy_together': {
      'type': 'array',
      'maxItems': 20,
      'items': {
```

A intenção era essa?